decidim-meetings/app/commands/decidim/meetings/admin/create_close_meeting_reminders.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Admin
+      class CreateCloseMeetingReminders < Decidim::Command
```

Comment:
The command and form are meant for purpose if you want to create an admin view to send the reminders manually.

You shouldn't need it when you don't implement the admin functionality.